rpcs3/Emu/RSX/VK/VKHelpers.h

```diff
@@ -1830,10 +1830,10 @@ namespace vk
 
 	class swapchain_X11 : public native_swapchain_base
 	{
-		Display *display = NULL;
+		Display *display = nullptr;
 		Window window = (Window)NULL;
-		XImage* pixmap = NULL;
-		GC gc = NULL;
```

Comment:
This is tricky and messes with readability. These types are from a C API which is why this doesn't work well with nullptr. For example: GC is an opaque type, its typedef into a pointer is not known/theoretically guaranteed. The type here is not even a pointer, so assigning nullptr to it is incorrect from a source code point of view. Ofc the compiler can see GC is typedefed into a pointer elsewhere which is why its suggesting this change.